Assets/Prefabs/Main Player health bar.prefab

```diff
@@ -0,0 +1,258 @@
+%YAML 1.1
```

Comment:
The health bar fill doesn't scale correctly horizontally with the outline. You can see this by making the Game window smaller horizontally. I think this is caused by the fill having a narrower width than the outline, but it's just a minor cosmetic problem. One solution would be to create and use a fill sprite with the same size as the outline sprite.